src/peer/base_node.js

```diff
@@ -0,0 +1,197 @@
+'use strict'
+
+const Swarm = require('libp2p-swarm')
+const TCP = require('libp2p-tcp')
+// const UTP = require('libp2p-utp')
+const WS = require('libp2p-websockets')
+const spdy = require('libp2p-spdy')
+const secio = require('libp2p-secio')
+const PeerId = require('peer-id')
+const PeerInfo = require('peer-info')
+const PeerBook = require('peer-book')
+const multiaddr = require('multiaddr')
+const mafmt = require('mafmt')
+const EE = require('events').EventEmitter
+
+exports = module.exports
+
+const OFFLINE_ERROR_MESSAGE = 'The libp2p node is not started yet'
```

Comment:
if we're defining strings like this, might as well use something like https://www.npmjs.com/package/i18n-2